internal/signaling/types.go

```diff
@@ -96,7 +96,8 @@ type ForwardablePacket struct {
 
 type CredentialsPacket struct {
 	cloudflare.Credentials
-	Type string `json:"type"`
+	Type      string `json:"type"`
+	RequestID string `json:"rid"`
```

Comment:
I would include the _omitempty_ option here, which removes the 'rid' if it's empty. Otherwise it'll always be there if it's an empty string.
```suggestion
	RequestID string `json:"rid,omitempty"`
```

Reply:
Done. I don't have a ton of context, but I'm wondering why the backend would ever want a credentials request without `rid` being set?